lib/datadog/lambda/metrics.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+#
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+#
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023 Datadog, Inc.
+#
+
+require 'datadog/statsd'
+
+module Datadog
+  # Metrics module contains the singleton class Client to send custom
+  # metrics to Datadog.
+  module Metrics
+    URL = 'localhost'
+    PORT = '8125'
+    # Client is a singleton class that instantiates a Datadog::Statsd
+    # client to send metrics to the Datadog Extension if present.
+    class Client
+      private_class_method :new
+
+      def self.instance
+        @instance ||= new
+      end
+
+      def distribution(name, value, time: nil, **tags)
```

Comment:
```suggestion
      def distribution(name, value, time: Time.now, **tags)
```

so you can remove the `time ||= Time.now` line above.

Reply:
Somehow, this makes the module `Time` to always return `nil` on `Time.now` in testing. So I'm going to skip it for now